index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div><a class="logo" href="#">My Logo</a></div>
+        <nav class="menu"><span class="material-icons">menu</span></nav>
+    </header>
+    <main>
+        <section class="main-content">
+            <h1 class="intro-text">I'm Islam Glad to see you!</h1>
+            <p class="main-text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+        <section class="footer">
+            <h4 class="end-text">LETS'S CONNECT</h4>
+            <ul class="social-conacts">
+                <li><i class="fa fa-twitter"></i></li>
+                <li><i class="fa fa-linkedin"></i></li>
+                <li><i class="fa fa-medium"></i></li>
+                <li><i class="fa fa-github"></i></li>
+                <li><i class="fa fa-angellist"></i></li>
```

Comment:
- [ ] I suggest you make the icons a link using the `<a>` tag. This makes it easier for anyone or companies to get in contact with you.